Uses refresh=false for creating, updating, and invalidating user sessions (#151800)
elastic/kibana: docs/user/security/session-management.asciidoc

If set, the value of this option should be an integer between `1` and `1000`.
When the limit is exceeded, the oldest session is automatically invalidated.

NOTE: Due to the rate at which session information is refreshed, there might be a few seconds where the concurrent session limit is not enforced.
This is something to consider for use cases where it is common to create multiple sessions simultaneously.

--
[source,yaml]
--------------------------------------------------------------------------------

> xpack.security.session.concurrentSessions:

Author: replaced cyrillic 'c' with latin 'c'

If set, the value of this option should be an integer between `1` and `1000`.
When the limit is exceeded, the oldest session is automatically invalidated.

> NOTE: Due to the rate at which session information is refreshed, there may be a small window of time, not longer than a few seconds, where the concurrent session limit is not enforced.

Reviewer: ```suggestion
NOTE: Due to the rate at which session information is refreshed, there might be a few seconds where the concurrent session limit is not enforced.
```
